### DISCUSSION TITLE
Docker Hub install: how do I get the "ozone" command in a shell?

### DISCUSSION THREAD

**Discussion by pjotrek-b** [apache/ozone#6721] [FAQ] [answered]
Hello everyone :smile: 

I'm truly sorry if I'm missing the obvious, but I'm trying to make a quick-n-dirty install of Ozone on a PVE virtual container and a single (virtual) disk.

Just to try it out.

I fail at the following:
  * Accessing the endpoint requires credentials.
  * Documentation says to run: `ozone s3 getsecret`

But I don't know how, since the VM OS has no `ozone` command installed. I presume it's inside the docker container. But how do I get a shell there?

I tried to find documentation about it, but failed.
I followed [these official (?) installation instructions](https://cwiki.apache.org/confluence/display/OZONE/Running+via+DockerHub) (and then consulted some others).

I'm grateful for any advice or links to docs. :cake:

**Comment by pjotrek-b** [comment, marked as answer]
I managed to get a shell inside the docker container (i renamed to) `ozone`:

```
$ docker start ozone
$ docker exec -it ozone sh
```

There I ran:
`sh-4.2$ ozone s3 getsecret`

Which gave me:
> Error: 'ozone s3 getsecret' operation works only when security is enabled. To enable security set ozone.security.enabled to true.

This looks like I shouldn't need any access credentials in the first place?
(like `ozone.security.enabled = false`by default?)

**Comment by pjotrek-b** [comment]
I know found this post (issue #5800), titled "How to create initial set of S3 credentials?", saying:

> "If security is not enabled you can set it as anything."

Thanks! That worked. I simply used `whatever/works` as access and secret keys with `aws configure`.

Then continuing to follow the official docs, I ran:
`$ aws s3api --endpoint http://localhost:9878/ create-bucket --bucket=bucket1`

Which simply froze. :sob:

**Comment by spacemonkd** [comment]
I know it is quite late, but we have a docker compose file at [here](https://github.com/apache/ozone/blob/master/hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml)
Here OZONE_RUNNER_IMAGE and OZONE_RUNNER_VERSION references images from [here](https://github.com/apache/ozone-docker-runner/pkgs/container/ozone-runner)

Maybe you could use the same to try it out.
